CHANGES_NEXT_RELEASE

```diff
@@ -14,4 +14,5 @@
 - Upgrade cobertura-maven-plugin from 2.6 to 2.7
 - Upgrade maven-surefire-report-plugin from 2.16 to 2.22.0
 - Upgrade maven-checkstyle-plugin from 2.12 to 3.0.0
-- Upgrade maven-javadoc-plugin from 2.9 to 3.0.1
\ No newline at end of file
+- Upgrade maven-javadoc-plugin from 2.9 to 3.0.1
+- Add: support for esper 6.1.0 timed rules("timer:XX" patterns and Match Recognize interval patterns)
```

Comment:
CNR line should mention the issue being solved. In addition a whitepsace whould be added before `(` (seems to be a typo). In sum, something like this:

```
 - Add: support for esper 6.1.0 timed rules("timer:XX" patterns and Match Recognize interval patterns)  (#91)
```

Reply:
Fixed in 589775e